www/wordpress/Portfile

```diff
@@ -3,68 +3,85 @@
 PortSystem          1.0
 
 name                wordpress
-version             4.9.10
+version             5.5.1
 revision            0
 
 categories          www
-platforms           darwin freebsd
-maintainers         nomaintainer
 license             GPL-2+
+maintainers         nomaintainer
+platforms           darwin freebsd
 supported_archs     noarch
 
-description         A state-of-the-art semantic personal publishing platform
-long_description    WordPress is a state-of-the-art semantic personal publishing platform \
-                    with a focus on aesthetics, web standards, and usability.
+description         a state-of-the-art semantic personal publishing platform
+long_description    WordPress is {*}${description} with a focus on aesthetics,\
+                    web standards, and usability.
 
 homepage            https://wordpress.org/
 master_sites        ${homepage}
 
-checksums           rmd160  96b9549cb6fb37c311c3c1973854f4e37b54a708 \
-                    sha256  359045880c4d49f600acb2c18808a4ba1d50090c1f759d930ee40e8173a77815 \
-                    size    8744264
+checksums           rmd160  3bf7930b3faa21a0f88395942b899c7397eeac68 \
+                    sha256  66a2ced5f3e240d0612dd3668fbd50308f9649ecb4a6e17d46dc0d8111b99295 \
+                    size    12983648
 
 worksrcdir          ${name}
 
+use_configure       no
+build               {}
+
 # The php variants deliberately do not conflict
-foreach php {php55 php56 php70 php71} {
-    variant ${php} description "Use ${php}" "
-        depends_run-append  port:${php}-gd \
-                            port:${php}-mbstring \
-                            port:${php}-mcrypt \
-                            port:${php}-mysql \
-                            port:${php}-zip
-    "
+#
+# min req is currently: 5.6
+#   » https://wordpress.org/news/2019/04/minimum-php-version-update/
+foreach v "56 70 71 72 73 74" {
+    variant php${v} description "Use php${v}" {
```

Comment:
Switching the variant code block from a double-quoted string `"..."` to a brace-quoted string `{...}` causes the `${v}` variable to be evaluated later -- when the variant code block is executed, which is long after the foreach loop is done -- rather than at each iteration of the loop as had been intended. Fixed in 928f34efd6031a6b55689f6f4f563dc6fc0955aa.